Solutions/CohesitySecurity/Tests/az.py

```diff
@@ -104,9 +105,54 @@ def get_subscription_id():
     return subscription_id
 
 
-def incident_show(incident_id, resource_group, workspace_name):
+def get_snow_system_ids(incident_details):
     """
-    Returns the details of a specific incident in JSON format.
+    This function takes in incident details as input, extracts the 'labels' from the incident details,
+    and searches for the 'SNOW System ID' label among the labels. If the label is found, it extracts
+    the alphanumeric string that represents the System ID from the label using a regular expression.
+    The function returns a list of the extracted SNOW System IDs.
+
+    :param incident_details: A dictionary containing incident details
+    :type incident_details: dict
+
+    :return: A list of SNOW System IDs extracted from the incident details
+    :rtype: list
+    """
+    incident_details_dict = incident_details
+    labels = incident_details_dict["labels"]
+    snow_system_ids = [
+        re.search(r"([a-fA-F0-9]{32})", label["labelName"]).group(1)
```

Comment:
Can you help me understand what this line of code is doing?

Reply:
done. add as comments in the codes.

Reply:
What's the reason the entire string is not used as is? Can you please post here in the review comment an example of what 'labels' look like?

Reply:
[
    {
        "labelName": "SNOW System ID: d3d64e4097122110a46a50081153af20",
        "labelType": "User"
    }
]

Reply:
[
    {
        "labelName": "SNOW System ID: d3d64e4097122110a46a50081153af20",
        "labelType": "User"
    }
]